rllib/agents/sac/sac_policy.py

```diff
@@ -12,10 +12,11 @@
 from ray.rllib.policy.tf_policy import TFPolicy
 from ray.rllib.policy.tf_policy_template import build_tf_policy
 from ray.rllib.models import ModelCatalog
-from ray.rllib.models.tf.tf_action_dist import SquashedGaussian, DiagGaussian
-from ray.rllib.utils.error import UnsupportedSpaceException
+from ray.rllib.models.tf.tf_action_dist import Categorical, SquashedGaussian, \
+    DiagGaussian
```

Comment:
pep8 seems to prefer implied linebreak with parentheses over backslashes: https://www.python.org/dev/peps/pep-0008/#maximum-line-length.
> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.

```suggestion
from ray.rllib.models.tf.tf_action_dist import (
    Categorical, SquashedGaussian, DiagGaussian)
```